terraform/storage/main.tf

```diff
@@ -0,0 +1,52 @@
+# Create a service running on fargate with a task definition and service definition
+terraform {
+  backend "s3" {
+    # Backend is selected using terraform init -backend-config=path/to/backend-<env>.tfbackend
+    # bucket         = "sdp-dev-tf-state"
+    # key            = "sdp-sandbox-tech-audit-tool-api-lambda/terraform.tfstate"
+    # region         = "eu-west-2"
+    # dynamodb_table = "terraform-state-lock"
+  }
+
+}
+
+# Remove the ECR repository creation since it already exists
+# Instead, use data source to reference existing ECR
+data "aws_ecr_repository" "tech_audit_tool" {
+  name = var.ecr_repository_name
+}
+
+# Initial ECR repository policy (without Lambda role)
+resource "aws_ecr_repository_policy" "tech_audit_tool_policy" {
```

Comment:
Why is this needed? The terraform doesn't publish the container to ECR does it?

Reply:
To be honest, there was a reason why I added this, however it was the other week so I have to refresh my memory as to why I did. I do remember that it was needed for some reason.

I have removed it though.